kokoro/config/test/third_party_apps/common.gcl

```diff
@@ -3,13 +3,5 @@ import '../common.gcl' as common
 template config third_party_apps_test = common.go_test {
   params {
     test_suite = 'third_party_apps_test'
-
-    environment {
-      // For presubmits, we want to pass -test.short, which will skip
-      // most app+distro pairs.
-      // TODO(martijnvans): Remove this setting from here once we've switched
```

Comment:
this setting lives (properly) in this file now: https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/test/third_party_apps/presubmit/common.gcl